.travis.yml

```
@@ -0,0 +1,17 @@
+language: ruby
+sudo: true
+
+rvm:
+ - 2.4.0
+
+services:
+ - postgresql
+
+before_script:
+ - cp ./config/database.yml.travis ./config/database.yml
+ - psql -c 'create database ls_test;' -U postgres
+ - rake db:migrate RAILS_ENV=test
+
+script:
+  - bundle exec rubocop --config .rubocop.yml
+  - bundle exec rspec
```

Will the build fail and `bundle exec rspec` be skipped if first step (Rubocop) fails?

---

The build will fail. 
`bundle exec rspec` will not be skipped.
https://circleci.com/gh/ZavyalovAndrey/link_shortener/48?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

---

Well, that's not good.. Please investigate options to fail fast when running a Travis build

---

https://travis-ci.org/ZavyalovAndrey/link_shortener/builds/275009030